security/expressions.rst

```diff
@@ -56,7 +56,9 @@ Additionally, you have access to a number of functions inside the expression:
     Returns ``true`` if the user is authenticated via "remember-me" or authenticated
     "fully" - i.e. returns true if the user is "logged in".
 ``is_anonymous``
-    Equal to using ``IS_AUTHENTICATED_ANONYMOUSLY`` with the ``isGranted()`` function.
+    Returns ``true`` if the user is anonymous. That is, the firewall confirms that it
+    does not know this user's identity. This is different from ``IS_AUTHENTICATED_ANONYMOUSLY``,
```

Comment:
What about renaming it to „IS_AUTHENTICATED_OR_ANONYMOUSLY“ to make it more clear and understandable?

cc @chalasr

Reply:
Sure, but that seems like a topic for its own PR?

Reply:
Absolutely this must be implemented int the code base first 👌🏻

Reply:
@OskarStark https://github.com/symfony/symfony/pull/31189 :)

Reply:
Thank you @chalasr 👍 

Could you approve this PR?

Reply:
It's not ready yet! This PR is correct for lower versions anyways